src/printer.js

```diff
@@ -477,8 +476,20 @@ function printExpression(path, options, print) {
         // @TODO: for now just reusing double/single quote preference from doc. could eventually
         // use setting from options. need to figure out how this works w/ complex strings and interpolation
         // also need to figure out splitting long strings
-        const quote = node.isDoubleQuote ? '"' : "'";
-        return makeString(stringEscape(node.value), quote, false);
+        let quote = node.isDoubleQuote ? '"' : "'";
+        let stringValue = node.raw;
+        if (path.getParentNode().kind === "encapsed") {
+          quote = "";
+        } else {
+          // if this is not part of an encapsed node, we need to strip out the quotes from the raw value
+          if (['"', "'"].includes(stringValue[0])) {
+            stringValue = stringValue.substr(1);
+          }
+          if (['"', "'"].includes(stringValue[stringValue.length - 1])) {
+            stringValue = stringValue.substr(0, stringValue.length - 1);
+          }
```

Comment:
I'm usually not too much into using Regexes, but here it might be shorter:
```js
if (['"', "'"].includes(stringValue[0])) {
  stringValue = str.replace(/^["'](.*)["']$/, '$1');
}
```

Reply:
🙏 thank you - im terrible with regex haha

Reply:
Me too, i basically copied this from StackOverflow :joy:

Reply:
hrm that doesn't seem to be working for me though

Reply:
Hmm, looks ok to me:
```
strip = str => str.replace(/^["'](.*)["']$/, '$1')
strip("\"foo\"")
"foo"
strip("'foo'")
"foo"
strip("'foo' a")
"'foo' a"
```

Reply:
for whatever reason this test is failing
```php
echo "This spans
multiple lines. The newlines will be
output as well";
```

Reply:
Oh, it was lacking multiline support. Try `^["']([\s\S]*)["']$`

Reply:
I think we're slowly reaching the point where the Regex becomes so cryptic that it's better to keep the longer, non-regex version :wink: